liquibase-core/src/main/java/liquibase/snapshot/jvm/ColumnSnapshotGenerator.java

```diff
@@ -526,6 +534,18 @@ protected Object readDefaultValue(CachedRow columnMetadataResultSet, Column colu
         return SqlUtil.parseValue(database, columnMetadataResultSet.get("COLUMN_DEF"), columnInfo.getType());
     }
 
+    /**
+     * {@link IndexSnapshotGenerator} fails to differentiate computed and non-computed column's for {@link PostgresDatabase}
+     * assume that if COLUMN_NAME contains parentesised expression -- its function reference.
+     * should handle cases like:
+     * - ((name)::text)
+     * - lower/upper((name)::text)
+     * - (name)::text || '- concatenation example'
+     */
+    private boolean looksLikeFunction(String columnName) {
+        return postgresFunctionRefPattern.matcher(columnName).find();
```

Comment:
Can there be functions that have empty parentheses setups we should b looking for too? Maybe should just check for `columnName.contains("(")`  and not bother with the regular expression?